tests/wallets/e2e/steps/helpers.js

```diff
@@ -86,8 +87,11 @@ export const restoreLegacyWallet = async (
           .refreshWalletsData()
           .then(() => {
             const wallet = daedalus.stores.wallets.getWalletByName(name);
-            if (transferFunds && wallet.amount.isZero()) {
-              throw new Error(noWalletsErrorMessage);
+            if (wallet) {
```

Comment:
@yakovkaravelov this is not ok. You need to throw an error if there is no wallet.